src/pages/LevelUpLandingPage/components/EventSection/EventSection.tsx

```diff
@@ -0,0 +1,50 @@
+import { subHours } from 'date-fns';
+import React from 'react';
+import { fetchEvents } from 'src/api/events';
+import FailedFetchAlert from 'src/components/FailedFetchAlert';
+import LoadableContent from 'src/components/LoadableContent';
+import EventSummary from 'src/components/EventSummary';
+import Loader from 'src/components/Loader';
+import { EventContainer, Link, Section } from './styled';
+import { formatDate } from 'src/utils/formatting';
+
+export default function EventSection() {
+  return (
+    <Section>
+      <h2>Upcoming Events</h2>
+
+      <LoadableContent
+        resolveContent={() => fetchEvents(subHours(new Date(), 12))}
```

Comment:
This fetch should probably be changed at some point to only fetch the next 3 events rather than fetching all the events and slicing the list.

Reply:
I think to only fetch 3 events requires an update to the API. 
Alternatively we could change the query to filter on i.e. all the events in the next month or 2 months?